### PR TITLE
Installation: add a mention of hyprland-meta package under Arch section

### PR DESCRIPTION
**Pull request by izmyname**
Add `yay -S  hyprland-meta-git` option to install the whole hypr* ecosystem, at once.

https://aur.archlinux.org/packages/hyprland-meta-git

### CODE REVIEW

**Comment by vaxerski** [comment]
what is the order in which the packages are compiled? That is important.

**Comment by izmyname** [comment]
hyprutils>hyprlang>hyprwayland-scanner>hyprland-protocols>hyprgraphics>aquamarine>xdph>hyprlock>hyprsunset>hyprland>hyprpaper>hyprpicker>hypridle>hyprpolkit>hyprsysinfo

**Comment by AlexanderCurl** [comment]
I have just tested the package on a clean Arch install, using Minimal Desktop profile in `archinstall`
Everything installed fine, manually installed paru from AUR, and used paru instead of yay, but whatever, should work fine.
If you using Hyprland desktop profile, it will break as expected, aquamarine has a pinned version for hyprland in the main repo, so first need to uninstall hyprland and all, then use the metapackage to install every git version.
I think this is good to go.

For hyprcursor, for me installed after aquamarine in the chain.

I haven't used any login manager, after installed hyprland-meta-git, rebooted the VM, and launched hyprland from the TTY, worked like a charm.

**Comment by izmyname** [comment]
Ah, yes, I forgot to mention: before writing my previous post - I wiped all hypr* packages along with aur cache and source dirs, then installed `hyprland-meta-git` on a clean hypr*-less system. Everything has compiled without errors and works, as it did before.

**Comment by vaxerski** [comment]
how is that enforced in the pkg?
image

This looks like hypridle is compiling before hyprland?

Anyways, the order you mentioned above looks ok

File: pages/Getting Started/Installation.md [hunk 1]
```diff
@@ -70,6 +70,12 @@ or a tagged release from the arch packages:
 sudo pacman -S hyprland
 ```
 
+Alternatively, install the `hyprland-meta` package to get the entire Hypr ecosystem.
```
Comment (vaxerski) [inline pages/Getting Started/Installation.md:73]: needs a mention that it will _compile_ and pull the latest git, which may not be ultra stable

Comment (izmyname) [reply pages/Getting Started/Installation.md]: Done

File: pages/Getting Started/Installation.md [hunk 2]
```diff
@@ -70,6 +70,12 @@ or a tagged release from the arch packages:
 sudo pacman -S hyprland
 ```
 
+Alternatively, install the `hyprland-meta` package to automatically fetch and compile the latest git versions of all components within the Hypr ecosystem.
```
Comment (vaxerski) [inline pages/Getting Started/Installation.md:73]: it's hypr* ecosystem not hypr ecosystem xd

Comment (izmyname) [reply pages/Getting Started/Installation.md:73]: Hmmm https://wiki.hyprland.org/Hypr-Ecosystem/

Comment (vaxerski) [reply pages/Getting Started/Installation.md]: I fear we can't put * in titles without something breaking

**Review by vaxerski — APPROVED** [review]

**Comment by izmyname** [comment]
Btw, looks like the site sync thingy is broken again.

**Comment by vaxerski** [comment]
thanks, fixed

**Pull request merged**